next-i18next.config.js

```diff
@@ -0,0 +1,36 @@
+// const getConfig = require('./tenant.config');
+const path = require('path');
+const LanguageDetector = require('i18next-browser-languagedetector');
+const { initReactI18next } = require('react-i18next');
```

Comment:
This does not seem to be used anymore. Probably these npm modules can be removed (also from package.json).